lib/facility_factory.rb

```diff
@@ -0,0 +1,44 @@
+class FacilityFactory
+
+  def initialize
+  end
+
+  def create_facility(dmv_info)
+    if dmv_info == @co_dmv_office_locations
```

Comment:
I'm unclear on what we are comparing dmv_info to here. There are no instance variables defined in this class, so the value of `@co_dmv_office_locations` is actually `nil` here. That means all of your data is running through your `else` logic flow. Your tests did not catch this because they're not specific enough, but if you put a binding.pry in your tests for this method, you'll see that you are getting back Facility objects that have no data filled in